### ISSUE TITLE
System.ArgumentOutOfRangeException     Message= Score column 'Score' not found Parametername: schema  

### ISSUE THREAD

**Issue by hattabatatta** [dotnet/docs#14854]
Followed this guide here -> https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices

Within the method Evaluate the error I mentioned above happens 
The exact line is as followed:
var metrics = mlContext.Regression.Evaluate(predictions, "Label", "Score");

edit by @mairaw: add topic details

[AB#1608693](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1608693)
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a30b3ff5-b5e6-ff02-6f60-b5cef6f34666
* Version Independent ID: fdf962e2-620e-d9ad-5057-5f96f52dc60f
* Content: [Tutorial: Predict prices using regression - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices)
* Content Source: [docs/machine-learning/tutorials/predict-prices.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/predict-prices.md)
* Product: **dotnet-ml**
* GitHub Login: @natke
* Microsoft Alias: **nakersha**

**Comment by luisquintanilla** [comment]
Hi @hattabatatta,

Sorry to hear you're having trouble building your application. After running the sample and re-creating the application using the tutorial I was not able to reproduce. 

A few things I would suggest:

1. Take a look at the source code used in this tutorial 
    https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/TaxiFarePrediction
1. Make sure that `TaxiFarePrediction` class is defined in the *TaxiTrip.cs* file.
    1. Make sure to use the `ColumnName` attribute with the value of **Score** for the `FareAmount` property.
1. Make sure that you have included all the steps in the pipeline.

```csharp
            var pipeline = mlContext.Transforms.CopyColumns(outputColumnName: "Label", inputColumnName: "FareAmount")
                .Append(mlContext.Transforms.Categorical.OneHotEncoding(outputColumnName: "VendorIdEncoded", inputColumnName: "VendorId"))
                .Append(mlContext.Transforms.Categorical.OneHotEncoding(outputColumnName: "RateCodeEncoded", inputColumnName: "RateCode"))
                .Append(mlContext.Transforms.Categorical.OneHotEncoding(outputColumnName: "PaymentTypeEncoded", inputColumnName: "PaymentType"))
                .Append(mlContext.Transforms.Concatenate("Features", "VendorIdEncoded", "RateCodeEncoded", "PassengerCount", "TripTime", "TripDistance", "PaymentTypeEncoded"))
                .Append(mlContext.Regression.Trainers.FastTree());
```

If you still continue to have issues, please feel free to open another issue.

**Comment by roarglobalca** [comment]
.Append(mlContext.Regression.Trainers.FastTree());
FastTree not available in 1.3.1 please advise what I should use.  Thank you

**Comment by luisquintanilla** [comment]
Hi @roarglobalca,

`FastTree` is a separate NuGet package. Please see the instructions in the tutorial under the *Create a console application* section.

> Install the Microsoft.ML NuGet Package: In Solution Explorer, right-click the project and select Manage NuGet Packages. Choose "nuget.org" as the Package source, select the Browse tab, search for Microsoft.ML, select the package in the list, and select the Install button. Select the OK button on the Preview Changes dialog and then select the I Accept button on the License Acceptance dialog if you agree with the license terms for the packages listed. **Do the same for the Microsoft.ML.FastTree Nuget package**.
